player/lua/console.lua

```diff
@@ -1754,7 +1754,7 @@ mp.observe_property('display-hidpi-scale', 'native', update)
 mp.observe_property('focused', 'native', update)
 
 mp.observe_property("user-data/osc/margins", "native", function(_, val)
-    if val then
+    if type(val) == "table" and type(val.t) == "number" and type(val.b) == "number" then
         global_margins = val
```

Comment:
What I was thinking of was something more like this:

```suggestion
        global_margins = { t = val.t or 0, b = val.b or 0 }
```

The idea is that a script can set only `b` or `t` and leave the rest unset and it'd (sensibly) default to 0.

Reply:
But thinking about it more, I think it's better to fix (usually) 1 writer than N readers. So documenting that all the l,r,t,b subproperties should be set is a good idea regardless of whether `console.lua` guards against it or not.